src/Couchbase.Lite.Shared/Support/Freezer.cs

```diff
@@ -58,6 +58,11 @@ public void SetValue<T>(ref T location, T newValue, [CallerMemberName]string cal
             location = newValue;
         }
 
+        public T GetValue<T>(ref T location)
+        {
+            return location;
```

Comment:
There is not really any reason to have this method, there is no harm in reading the variable directly

Reply:
Same reason as below. I want to retrieve the stored frozen Continuous value for internal test.

Reply:
In the end all you are doing is returning `_continuous`.  The Freezer does not store anything, it's just a gatekeeper.

Reply:
All this is doing is returning `_continuous` though, Freezer doesn't actually store anything, it's just returning the same thing that got passed in as the argument.

Reply:
Removed.